screens/SendSetupScreen.js

```diff
@@ -287,7 +295,9 @@ const SendSetupScreen = ({
 
   const sendAmountNumber = parseFloat(sendAmount);
 
-  const coinName = !tokenId ? "Bitcoin Cash" : tokensById[tokenId].name;
+  const tokenName =
```

Comment:
`SLP Token` may be an actual tokens name, for undefined values I tend to prefer sticking with `undefined` then special casing it in the render logic if it doesn't exist.  or defaulting to something clearly not real such as `----`